Modules/src/Logging.cc

```diff
@@ -64,6 +64,25 @@ void Logger::sendMessage(const std::string& msg, const logging::LogLevel& level)
   }
 }
 
+void Logger::prepare(){
+  // write initial value in order to bring LoggingModule to mainLoop()
+  message.write();
+}
+
+LoggingModule::LoggingModule(ctk::EntityOwner* owner, const std::string& name, const std::string& description,
+      ctk::HierarchyModifier hierarchyModifier,
+      const std::unordered_set<std::string>& tags):
+        ApplicationModule(owner, name, description, hierarchyModifier, tags){
+  auto virtualLogging = getOwner()->findTag("Logging");
```

Comment:
You are merely searching for all variables tagged with "Logging" in the owner, not in the entire application. I think, this is fine, because it allows even more control (you could add a LoggingModule at a deeper place in the hierarchy to only log what is coming from below that point in hierarchy). You should just be aware of this and correct the comment above.

Also I would recommend not to hardcode the tag name, but instead pass it as a constructor argument. This also would allow more control, e.g. if different logging sources should be logged differently in separate LoggingModules.